src/reporting/reportingToken.se

```diff
@@ -81,8 +80,26 @@ def migrateLosingTokens():
     require(self.getBranch().getForkingMarket() != self.market)
     require(self.market.getFinalWinningReportingToken() != self)
     self.supply = 0
-    reputationSupply = self.getReputationToken().balanceOf(self)
-    self.getReputationToken().transfer(self.market.getFinalWinningReportingToken(), reputationSupply)
+    # Send losing reporting token REP to correct dispute bonds until bond holders are paid out (either all REP are sent or they hit 2x limit)
+    # Pay out to earlier disputers as much as possible before paying out to later disputers
+    reputationToken = self.getReputationToken()
+    automatedReporterDisputeBondToken = self.market.getAutomatedReporterDisputeBondToken()
+    limitedReportersDisputeBondToken = self.market.getLimitedReportersDisputeBondToken()
+    allReportersDisputeBondToken = self.market.getAllReportersDisputeBondToken()
+    if (automatedReporterDisputeBondToken and automatedReporterDisputeBondToken.getDisputedPayoutDistributionHash() != self.market.getFinalPayoutDistributionHash()):
+        amountNeeded = automatedReporterDisputeBondToken.getBondRemainingToBePaidOut() - reputationToken.balanceOf(automatedReporterDisputeBondToken)
+        amountToTransfer = min(amountNeeded, reputationToken.balanceOf(self))
+        if (amountToTransfer > 0):
+            reputationToken.transfer(automatedReporterDisputeBondToken, amountToTransfer)
+            log(type=Transfer, self, automatedReporterDisputeBondToken, amountToTransfer)
```

Comment:
We aren't transferring reporting tokens here, so this log is unnecessary and should be avoided.  There may be value in logging _some_ event when this happens, but it wouldn't be the `Transfer` event.  For now, I recommend just deleting this log.

Same feedback for other log line below.